packages/tools/locales/nl_NL.po

```diff
@@ -3302,9 +3301,8 @@ msgid "Downloaded and encrypted"
 msgstr "Gedownload en versleuteld"
 
 #: packages/lib/services/ReportService.js:180
-#, fuzzy
 msgid "Created locally"
-msgstr "Lokale items gecreëerd: %d."
+msgstr "Lokale items gecreëerd: %d"
```

Comment:
The source does not have a `%d`. Why does the translation? Something is wrong here.

Reply:
The source does have "Created local items: %d."

Reply:
The source string for this translation is `Created Locally` (line 3304).

The translation you are talking about is on line 2994.

Reply:
In Poedit search for the source string `Created locally` and you'll see what I mean.

Reply:
No worries. Just make the changes in the other PR, add the commit and I can merge...